io.catenax.shared.business_partner_certificate/2.0.0/BusinessPartnerCertificate.ttl

```diff
@@ -0,0 +1,173 @@
+
+ :validUntil a samm:Property;
+    samm:preferredName "Valid until"@en;
+    samm:description "Valid valid until as defined on the certificate. If certificate never expires value until expected to be 9999-12-31"@en;
+    samm:characteristic :Date;
+	samm:exampleValue "2026-01-24"^^xsd:date.
+    
+ :Date a samm:Characteristic ;
+   samm:preferredName "Date"@en ;
+   samm:description "Describes a property which contains the date in english format."@en ;
+   samm:dataType xsd:date;
+   samm:exampleValue "2026-01-24"^^xsd:date.
+  
+:validator a samm:Property ;
+   samm:preferredName "Validator"@en ;
+   samm:description "The BPN of the data service provider who validate the given certificate"@en ;
+   samm:characteristic :TrustValidatorCharacteristic.
+
+:TrustValidatorCharacteristic a samm:Characteristic;
+	samm:preferredName "Validiator caracteristic"@en;
+	samm:description "The BPN of the data service provider who validated the given certificate"@en ;
+    samm:dataType :TrustValidatorEntitity . 
+   
+   
+:TrustValidatorEntitity a samm:Entity ;
+   samm:preferredName "Trust validator entity"@en ;
+   samm:description "The BPN of the data service provider who validated the given certificate"@en ;
+   samm:properties ([samm:property :validatorName; samm:optional true] [samm:property :validatorBpn; samm:optional true]).
+
+:validatorName a samm:Property ;
+   samm:preferredName "Validator name"@en ;
+   samm:description "The optional name of the data service provider who validated the given certificate"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Data service provider X".
+
+:validatorBpn a samm:Property ;
+   samm:preferredName "Certifcate validator Bpn"@en ;
+   samm:description "The BPN of the data service provider who validated the given certificate"@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL00000007YREZ".
+ 
+ :issuer a samm:Property ;
+   samm:preferredName "Issuing authority"@en ;
+   samm:description "The BPN of the issuing authority e.g. TUEV Sued "@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL00000023ZAVC".
+
+ :trustLevel a samm:Property;
+	samm:preferredName "Trust level"@en ; 
+	samm:description "The trust level of the given certificate - none,low, high, trusted"@en ;
+	samm:characteristic :TrustLevelValue ;
+	samm:exampleValue "high".
+	
+ :TrustLevelValue a samm-c:Enumeration;
+   samm:preferredName "Trust level value"@en ;
+   samm:description "The possible trust level values of certificate"@en ;
+   samm:dataType xsd:string;
+   samm-c:values ("none" "low" "high" "trusted").
+  
+ 
+ :uploader a samm:Property ;
+   samm:preferredName "Certifcate uploader"@en ;
+   samm:description "The BPN of the business partner who originally provided the certifcate data or document e.g. Mercedes Benz AG"@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL00000003AYRE".
+
+
+ :documentID a samm:Property ;
```

Comment:
@SujitMBRDI is this going to be a unique UUID? if yes, you can make use of shared UUID model here instead of making it a text field.

Reply:
Hi @agg3fe, 
No, we are sticking to pervious version approach here. As per the Christian @chr-krenn, it is not necessary to define a global UUID here from a semantic model perspective.